pontoon/machinery/utils.py

```diff
@@ -61,6 +79,45 @@ def get_google_translate_data(text, locale_code):
         }
 
 
+def get_google_automl_translation(text, locale):
+    try:
+        client = translate.TranslationServiceClient()
+    except DefaultCredentialsError as e:
+        log.error("Google AutoML Translation error: {e}")
+        return {
+            "status": False,
+            "message": f"{e}",
+        }
+
+    project_id = "85591518533"
```

Comment:
This magic constant should probably be a well commented config value, yes?

Reply:
Of course, nice catch!